docker/Dockerfile

```
@@ -0,0 +1,31 @@
+## First stage: build the application
+FROM golang:1.13.0-alpine AS builder
+ARG GOARCH
+ARG GOARM
+
+WORKDIR /go/src/github.com/CESARBR/knot-babeltower
+COPY . .
+
+# Install glide dependency manager
+RUN apk update \
+    && apk add make git curl \
+    && curl https://glide.sh/get | sh
+
+# Install project development tools and dependencies
+RUN make tools
+RUN make deps
+
+# Build the application
+RUN make bin
+
```

Regarding the review made prior to this one, remove the virtual package created (with the build dependencies).

```suggestion
RUN apk del .build-deps

```

---

Done.